python/lancedb/table.py

```diff
@@ -341,16 +341,18 @@ def checkout(self, version: int):
             raise ValueError(f"Invalid version {version}")
         self._reset_dataset(version=version)
 
-    def restore(self, version: int):
+    def restore(self, version: int = None):
         """Restore a version of the table. This is an in-place operation.
 
         This creates a new version where the data is equivalent to the
-        specified previous version. Note that this creates a new snapshot.
+        specified previous version. Data is not copied (as of python-v0.2.1).
 
         Parameters
         ----------
-        version : int
-            The version to restore.
+        version : int, default None
+            The version to restore. If unspecified then restores the currently
+            checkedout version. If the currently checkedout version is the latest
+            version then this is a no-op.
```

Comment:
```suggestion
            checked out version. If the currently checked out version is the
            latest version then this is a no-op.
```
I'm not used to seeing "checkedout" as one word but if its a convention feel free to ignore this.

Reply:
oh that's a total typo. thanks for flagging it